src/post.rs

```diff
@@ -83,3 +83,45 @@ pub fn relink(
 
     Ok(())
 }
+
+/// Find any .dist-info/INSTALLER files and replace the contents with "conda"
+/// This is to prevent pip from trying to uninstall the package when it is installed with conda
+pub fn python(
+    name: &PackageName,
+    version: &str,
+    paths: &HashSet<PathBuf>,
+) -> Result<(), std::io::Error> {
+    let metadata_glob = globset::Glob::new("**/*.dist-info/METADATA")
+        .unwrap()
+        .compile_matcher();
+
+    if let Some(p) = paths.iter().find(|p| metadata_glob.is_match(p)) {
+        // unwraps are OK because we already globbed
+        let distinfo = p
+            .parent()
+            .unwrap()
```

Comment:
Please use expect instead of unwrap with a message on why this should never fail. That makes your assumption more explicit